Added settings-profile-navbars.html to the navbars folder

oluscc opened the pull request:
This navbar updated the general navbar by adding a drop down menu for settings containing the following links: "Profile Settings", "Email Settings" and "Subscriptions".

With this navbar on all the pages for a signed in user, we have a consistent look for the navigations and a signed in user can access the settings  for profile, email and subscription from any of the pages.

Using the navbar on the /dashboard/profile/settings page will replace the current navbar with the general one being used on all the pages for consistent look.

The update navbar when added to the /dashboard/profile/settings page look like the image below:

 ![settings-profile_oluscc](https://user-images.githubusercontent.com/52763746/67383819-dbe32f80-f587-11e9-87f8-7118075de219.png)

Instead of the current look below which is not consistent with the other pages.

![settings-profile_current](https://user-images.githubusercontent.com/52763746/67383817-dbe32f80-f587-11e9-97b2-fddd424937f0.png)

zonebyte commented:
This is the updated Invoice view page.

It is online at zonebyte.github.io/invoi

![Capture.PNG](https://cdn.steemitimages.com/DQmb5KnLYLwXyi7W2P9kdm7oCv36t19LQ6Ts9q7gCzUVkri/Capture.PNG)

beardedprince commented:
Good one bro
 Please kindly remove the hamburger menu on desktop view..

And follow the design pattern we talked about

OluSmartDev committed:
Merge pull request #1 from hngi/dev

Update Forked Repo

OluSmartDev committed:
Create settings-profile-navbars.html

Added Drop Down Menu to the 'Settings' contaning the following links:  Profile Settings, Email Settings and Subscriptions.
